pkg/nsx/services/vpc/vpc.go

```diff
@@ -389,22 +389,48 @@ func (s *VPCService) GetAVISubnetInfo(vpc model.Vpc) (string, string, error) {
 	return path, cidr, nil
 }
 
-func (s *VPCService) GetNSXLBSNATIP(vpc model.Vpc) (string, error) {
+func (s *VPCService) GetNSXLBSNATIPForCTGW(vpc model.Vpc) (string, error) {
 	log.V(2).Info("Getting VPC NSX LB SNAT IP", "VPC", *vpc.Id)
 	_, err := common.ParseVPCResourcePath(*vpc.Path)
 	if err != nil {
 		return "", err
 	}
 
 	realizeService := realizestate.InitializeRealizeState(s.Service)
-	tier1UpLinkIP, err := realizeService.GetPolicyTier1UplinkPortIP(*vpc.Path)
+	parts := strings.Split(*vpc.Path, "/vpcs/")
+	if len(parts) != 2 {
```

Comment:
This check is redundant because this check is already done by common.ParseVPCResourcePath

Reply:
fixed